docs/src/modules/components/DemoToolbar.js

```diff
@@ -673,6 +673,9 @@ export default function DemoToolbar(props) {
       >
         <MenuItem
           value={CODE_STYLING.SYSTEM}
+          data-ga-event-category="demo"
+          data-ga-event-action="styling-system"
```

Comment:
Should we update

```jsx
const CODE_STYLING = {
  SYSTEM: 'MUI System',
  TAILWIND: 'Tailwind',
  CSS: 'CSS',
};
```

to match?

Reply:
I tried to follow how the other actions look like - they are all kebab-case.